gordo_components/builder/build_model.py

```diff
@@ -192,6 +189,7 @@ def build_model(
     }
 
```

Comment:
Perhaps remove this line if there is to be a newline between metadata and the return statement :)

Reply:
Sweet. Thanks Einar!